packages/core/src/factory.ts

```diff
@@ -90,53 +73,59 @@ export function factory<Resource, Input, Output, Config extends IPrismConfig>(
                     data: input,
                   },
                   config,
-                }),
-                TaskEither.map(output => ({ output, resource })),
-              );
-            }
-
-            return TaskEither.left(new Error('Nor forwarder nor mocker has been selected. Something is wrong'));
-          }),
-          TaskEither.map(({ output, resource }) => {
-            let outputValidations: IPrismDiagnostic[] = [];
-            if (resource && components.validator && components.validator.validateOutput) {
-              outputValidations = components.validator.validateOutput({
+                })(components.logger.child({ name: 'NEGOTIATOR' })),
+              ),
+              TaskEither.map(output => ({ output, resource })),
+            );
+          } else if (components.forwarder) {
+            // forward request and set output from response
+            return pipe(
+              components.forwarder.fforward({
                 resource,
-                output,
+                input: {
+                  validations: {
+                    input: inputValidationResult,
+                  },
+                  data: input,
+                },
                 config,
-              });
-            }
+              }),
+              TaskEither.map(output => ({ output, resource })),
+            );
+          }
 
-            return {
-              input,
```

Comment:
Neither mock or forwarder

Not doesn’t work quite like that :)

Reply:
I do not understand what you're trying to say

Reply:
new Error('Neither forwarder or mocker have been selected. Something is wrong'));